README.md

```diff
@@ -12,5 +49,9 @@ To use, add this code to your index.ios.js and index.android.js (or some library
 ```
 // Already assumes that Fabric is initialized/configured properly in the iOS and Android app startup code.
 import crashlytics from 'react-native-fabric-crashlytics';
-crashlytics.init();
-```
\ No newline at end of file
+function crashload () {
+  const path = `sourcemap.js`
+  crashlytics.init(path)
```

Comment:
I don't understand why you did this function, versus:
`crashlytics.init('sourcemap.js')`

Reply:
Ah, that's because while I was trying to get it working well, I went through so many versions that I forgot to change that back to a simple way